docs/standard/frameworks.md

```diff
@@ -28,11 +25,11 @@ For package-based target frameworks (for example, .NET Standard and .NET Core),
 
 The following table defines the most common target frameworks, how they're referenced, and which version of the [.NET Standard](~/docs/standard/net-standard.md) they implement. These target framework versions are the latest stable versions. Pre-release versions aren't shown. A Target Framework Moniker (TFM) is a standardized token format for specifying the target framework of a .NET app or library. 
 
-| Target Framework      | Latest Version | Target Framework Moniker (TFM) | .NET Standard Version | Metapackage |
-| :-------------------: | :------------: | :----------------------------: | :-------------------: | :---------: |
-| .NET Standard         | 2.0.0          | netstandard2.0                 | N/A                   | [NETStandard.Library](https://www.nuget.org/packages/NETStandard.Library) |
-| .NET Core Application | 2.0.0          | netcoreapp2.0                  | 2.0                   | [Microsoft.NETCore.App](https://www.nuget.org/packages/Microsoft.NETCore.App) |
-| .NET Framework        | 4.7            | net47                          | 1.5                   | N/A |
+| Target Framework      | Latest Version | Target Framework Moniker (TFM)   | Minimum <br/> .NET Standard Version | Metapackage |
+| :-------------------: | :------------: | :------------------------------: | :---------------------------: | :---------: |
+| .NET Standard         | 2.0.0            | netstandard2.0                 | N/A                           | [NETStandard.Library](https://www.nuget.org/packages/NETStandard.Library) |
+| .NET Core Application | 2.0.0            | netcoreapp2.0                  | 2.0                           | [Microsoft.NETCore.App](https://www.nuget.org/packages/Microsoft.NETCore.App) |
+| .NET Framework        | 4.7            | net47                            | 1.0                           | N/A |
```

Comment:
.NET Framewor 4.7 also implements .NET Standard 2.0.

Reply:
What I'm trying to say is the lowest version of the standard that you can target. That's why I went with minimum. We could also remove that column too.